playout/install/logrotate/libretime-liquidsoap.conf

```diff
@@ -1,4 +1,4 @@
-/var/log/airtime/pypo-liquidsoap/ls_script.log {
+/var/log/libretime/liquidsoap.log {
```

Comment:
We need to mention the change in logging directory in the release notes for beta.1/alpha.11 whichever comes first

Reply:
This should go in the commit message, I'll add the BREAKING change notice in the PR description, it should be be present in the changelog once we generate it.

Reply:
Even better